AlphaWallet/WalletConnect/Types/EIP712TypedData.swift

```diff
@@ -78,8 +78,9 @@ extension EIP712TypedData {
         var depSet = findDependencies(primaryType: primaryType)
         depSet.remove(primaryType)
         let sorted = [primaryType] + Array(depSet).sorted()
-        let encoded = sorted.map { type in
-            let param = types[type]!.map { "\($0.type) \($0.name)" }.joined(separator: ",")
+        let encoded = sorted.compactMap { type in
```

Comment:
Is this because there is an unsupported type that we should support?

Reply:
i am not quite sure that type is not supported, when we calculate `digest` for typedData we check for `EIP712Domain` that is missing for typed data received from Wallet Connect v2, as i know it should present there

Reply:
Hmm.. OK, can you help leave a comment there before the compactMap to mention the problem we are avoiding? So when we see the code (and the comment) the next time we know?

Reply:
Normally I'd rely on `git log -S`, but seems hard to think of searching git log for this :)